qubesadmin/tests/mock_app.py

```diff
@@ -474,6 +549,16 @@ def __init__(self, qapp):
                     b'2\x00QubesNoSuchPropertyError\x00\x00No such property\x00'
                 continue
 
+        for property_name, value in DEFAULT_DOM0_PROPERTIES.items():
+            self.qapp.expected_calls[(self.name, "admin.property.Get",
+                        property_name, None)] = \
+                b"0\x00" + str(value).encode()
+
+    def set_global_property(self, property_name: str, value: str):
```

Comment:
It's a bit weird to set global properties via dom0 (same applies to the look above) - normally those are properties of `Qubes()` object directly, not `Qubes().domains[0]`. But if that's more convenient for some reason, I guess it's fine for tests.